### ISSUE TITLE
Crash (stack dump) upon: Error: Type "AWSDateTime" not found in document.

### ISSUE THREAD

**Issue by berenddeboer** [cjoudrey/graphql-schema-linter#204]
When trying out this module it correctly reports about a type I haven't defined, but then seems to crash??

```
node_modules/graphql-schema-linter/lib/cli.js ../schema/cms.graphql.tmp
It looks like you may have hit a bug in graphql-schema-linter.

It would be super helpful if you could report this here: https://github.com/cjoudrey/graphql-schema-linter/issues/new

Error: Type "AWSDateTime" not found in document.
    at ASTDefinitionBuilder._resolveType (node_modules/graphql/utilities/buildASTSchema.js:100:11)
    at ASTDefinitionBuilder.buildType (node_modules/graphql/utilities/buildASTSchema.js:210:79)
    at ASTDefinitionBuilder._buildWrappedType (node_modules/graphql/utilities/buildASTSchema.js:229:17)
    at ASTDefinitionBuilder.buildField (node_modules/graphql/utilities/buildASTSchema.js:249:18)
    at node_modules/graphql/utilities/buildASTSchema.js:332:21
    at node_modules/graphql/jsutils/keyValMap.js:36:31
    at Array.reduce (<anonymous>)
    at keyValMap (node_modules/graphql/jsutils/keyValMap.js:35:15)
    at ASTDefinitionBuilder._makeFieldDefMap (node_modules/graphql/utilities/buildASTSchema.js:329:48)
    at fields (node_modules/graphql/utilities/buildASTSchema.js:312:22)
```

So in case if anyone is interested in this report.

**Comment by cjoudrey** [comment]
@berenddeboer what version of `graphql-schema-linter` are you experiencing this issue with?

I'm asking because I can't reproduce this.

```
$ cat test.graphql
type Query {
  a: MyType
}

$ graphql-schema-linter --version
0.2.4

$ graphql-schema-linter test.graphql
test.graphql
2:6 Unknown type "MyType".  invalid-graphql-schema

✖ 1 error detected
```

**Comment by berenddeboer** [comment]
0.2.3. With simple schemes I can't repeat it either. But I can with the attached scheme:

```
node_modules/graphql-schema-linter/lib/cli.js tmp.graphql
```

See attached file:
[graphql.txt](https://github.com/cjoudrey/graphql-schema-linter/files/4125733/graphql.txt)

**Comment by berenddeboer** [comment]
Just tried 0.2.4, same crash.

**Comment by cjoudrey** [comment]
Thanks!

That full schema should help with debugging the issue.

In the meantime, here's a workaround for a similar issue:
https://github.com/cjoudrey/graphql-schema-linter/issues/165#issuecomment-468379498

**Comment by cjoudrey** [comment]
Sorry for the delay. Just circling back to this issue. 

I ran the linter against `graphql.txt` and it worked fine. 🤔 

```
$ graphql-schema-linter graphql.txt
graphql.txt
14:12 Unknown type "AWSDateTime".  invalid-graphql-schema
15:12 Unknown type "AWSDateTime".  invalid-graphql-schema
22:12 Unknown type "AWSDateTime".  invalid-graphql-schema
31:12 Unknown type "AWSDateTime".  invalid-graphql-schema
36:12 Unknown type "AWSDateTime".  invalid-graphql-schema
37:12 Unknown type "AWSDateTime".  invalid-graphql-schema
45:12 Unknown type "AWSDateTime".  invalid-graphql-schema
46:12 Unknown type "AWSDateTime".  invalid-graphql-schema
```

I couldn't help but notice `node_modules/...` in the command you pasted. Do you happen to have `graphql-js` in `node_modules/...` as well? It makes me wonder if maybe the problem is caused by a newer version of GraphQL.js which I haven't tested.
